src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -220,7 +219,7 @@ export default {
 			return _without(this.categoryIds, ...this.customCategoryIds);
 		},
 		leadHeaderFilterLink() {
-			return this.lendFilterExpVersion === 'b' ? '/lend/filter' : '/lend';
+			return '/lend';
```

Comment:
This should be the other way around. We should not be intentionally directing traffic into legacy `/lend` unless there are a subset of query params that can't be fulfilled by the `/lend/filter` page. Seeing as this is a clean link, it should be `/lend/filter`.

Reply:
@mcstover Will do that change, thank you very much for correcting me! 👍

Reply:
It's ok, this would generally make sense but the lend_by_category_V2 exp is tightly coupled across the UI app and our legacy app and actually originates here: https://github.com/kiva/kiva/blob/development/httpdocs/classes/API/Experiment/LendFilter.php

We need to do a bit more analysis on what removing this UI code does as we don't want to move back to a world where the legacy page gets traffic by default.

We can move this convo to slack to get into the details.

FYI @emuvente

Reply:
To be clear, this is about lend_filter_v2 not lend_by_category_v2